src/content/post/2025-07-14-kuzu-v-0.11.0.md

```diff
@@ -34,12 +33,11 @@ is added to the graph. Once an index is created on a table, any further insertio
 in that table will automatically trigger an update in the associated index.
 
 If your workflows involve manually dropping and recreating indices on tables that are updated
-with new data, now, you can just focus on inserting the data, and Kuzu will take care of the rest.
+with new data, you can now just insert the data, and Kuzu will handle the rest.
 
 ### Filtered vector search with arbitrary Cypher queries
 
-Our initial version of vector search provided the ability to perform filtered search with simple predicates
-on a single table. In v0.11.0, we generalize the predicate to be an arbitrary Cypher query. Consider the following
+Earlier versions of vector search only supported simple, single-table filters. In v0.11.0, filters can now be expressed as arbitrary Cypher queries. Consider the following
```

Comment:
```suggestion
Earlier versions of vector search only supported simple predicate-based filters on a given table. In v0.11.0, we generalize this so that filters can be expressed as arbitrary Cypher queries. Consider the following
```